packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Models/Types/MrwSerializationTypeProvider.cs

```diff
@@ -51,6 +52,8 @@ public MrwSerializationTypeProvider(ModelProvider model, InputModelType inputMod
             Namespace = model.Namespace;
         }
 
+        protected override string GetFileName() => Path.Combine("src", "Generated", "Models", $"{Name}.serialization.cs");
```

Comment:
So could this be a property? It feels like a property instead of an action.

Reply:
All of our properties are implemented this way.  They are lazy loaded with a nullable field and overridden by a protected get method.

Reply:
Oh i see - i thought this is the property.
Then it is fine.